homework37/src/App.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react'
+import { v4  } from 'uuid';
+import TodoForm from './components/TodoForm'
+import TodoList from './components/TodoList'
+type Todo = {
+  title: string 
+  id: number 
+  isCompleted: boolean 
+}[]
```

Comment:
```suggestion
type Todo = {
  title: string 
  id: number 
  isCompleted: boolean 
}
```
мені ріже око, що назва типу в однині а по факту це масив
добре коли назва типу чи змінної для масиву називається у множині
але тут краще все ж таки визначити тип для однієї сутності